documentation/inventory_file_v2.md

```diff
@@ -304,12 +307,19 @@ groups:
       transport: ssh
   - name: windows
     targets:
-      - uri: target3.example.com
-        alias: windows1
-      - uri: target4.example.com
-        alias: windows2
+      - name: win1
+        uri: target3.example.com
+      - name: win2
+        uri: target4.example.com
     config:
       transport: winrm
+config:
+  ssh:
```

Comment:
It seems useful to set one of these at the group level as well so people know that you can set more than just `transport` there and also to demo that the transport needs to be a key under `config`.